app/models/borrow_history.rb

```diff
@@ -0,0 +1,36 @@
+class BorrowHistory < ApplicationRecord
+
+  belongs_to :user
+  belongs_to :book
+  enum status: {
+    pending: 'pending',
```

Comment:
enums should have value of integer as you have gained nothing from using keys with values the same